inc/Integrations/Airtable/AirtableDataSource.php

```diff
@@ -96,4 +96,67 @@ public function to_ui_display(): array {
 			'uuid'    => $this->config['uuid'] ?? null,
 		];
 	}
+
+	public function ___temp_get_query(): AirtableGetItemQuery|\WP_Error {
```

Comment:
Is the naming here to indicate this isn't how we want this API to look in the future?